tests/test_mpscircuit.py

```diff
@@ -305,28 +303,39 @@ def test_MPO_conversion(highp, tfb):
     np.testing.assert_allclose(tensor4, tensor4_ref, atol=1e-12)
 
 
-@pytest.mark.parametrize(
-    "backend, dtype", [(lf("tfb"), lf("highp")), (lf("jaxb"), lf("highp"))]
-)
+@pytest.mark.parametrize("backend, dtype", [(lf("tfb"), lf("highp"))])
```

Comment:
the jax backend is turned off since it is slow?

Reply:
Yes, only because it's slow (both with and without jit)

Reply:
> Yes, only because it's slow (both with and without jit)

Then I suggest to use a small N and D for the test file, so that at least functionality for both backend can be checked?

Reply:
OK